### ISSUE TITLE
[Feature request] Generate runners per tag

### ISSUE THREAD

**Issue by GrecPetroff** [temyers/cucumber-jvm-parallel-plugin#48]
Hey @temyers , 

Are there any possibilities to generate runners per tags in feature?
That will be another workaround of splitting scenarios on parts.

Like:

```
  Scenario Outline: func test
    Given some <data>
    And test func
    
  @theTagPart2
    Examples:
    |data|
    |any|

  @theTagPart1
    Examples:
    |data|
    |some|
```
As result we will have 2 runners with 2 different tags.

**Comment by sugatmankar** [comment]
@temyers  please review tagwise branch https://github.com/sugatmankar/cucumber-jvm-parallel-plugin/tree/tagwise. 

I have created tagwise runner it .

@GrecPetroff we are in progress to develop this for Scenario Outline.

**Comment by sugatmankar** [comment]
[tagwiseOutlinewiseIssueRerun]( https://github.com/sugatmankar/cucumber-jvm-parallel-plugin/tree/tagwiseOutlinewiseIssueRerun) Branch has done implementation of outline parallelization.

**Comment by temyers** [comment]
Fixed in 2.0.0 - a new Scenario runner generates a runner for every scenario/ScenarioOutline example.  Tags then filter the scenarios to be executed.  Only scenarios with matching tags should get created.
